modules/deepintentDpesIdSystem.md

```diff
@@ -0,0 +1,49 @@
+# Deepintent DPES ID
+
+The Deepintent DPES ID or Healthcare Identity is a shared, neutral identifier that publishers and ad tech platforms can use to recognise users even in environments where 3rd party cookies are not available. The Deepintent DPES ID is designed to respect users' privacy choices and publishers’ preferences throughout the advertising value chain. 
+
+## Deepintent DPES ID Registration
+
+The Deepintent DPES ID is free to use, but requires a simple registration with Deepintent. Please contact your sales rep to get started
+
+
+## Deepintent DPES ID Configuration
+
+First, make sure to add the Deepintent submodule to your Prebid.js package with:
+
+```
+gulp build --modules=deepintentDpesIdSystem,userId
+```
+
+The following configuration parameters are available:
+
+```javascript
+pbjs.setConfig({
+  userSync: {
+    userIds: [{
+      name: 'deepintentId',
+      params: {
+        siteId: 173,            // change to the siteId number to the one you recieved from Deepintent.
+        identityKey: "hashedEmail" // HashedEmail or HashedNPI based on type of your integration
+      },
+      storage: {
+        type: 'cookie',           // "html5" is the required storage type option is "html5"
+        name: '_di',            // change to the cookie name you wish to read from, optional if storage type is html5
```

Comment:
why do you treat cookie and html5 differently?

Reply:
We give a way to specify cookie name for pubs and for localstorage key is already set with us. This is to accomodate the exisitng solution we have. Once we made changes to that code we might change the working here as well for now no changes and thank you for suggestion.

Reply:
Changes are completed as suggested.